herwigpp-toolfile.spec

```diff
@@ -11,12 +11,16 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/herwigpp.xml
 <tool name="herwigpp" version="@TOOL_VERSION@">
   <client>
     <environment name="HERWIGPP_BASE" default="@TOOL_ROOT@"/>
-    <environment name="LIBDIR" default="$HERWIGPP_BASE/lib"/>
-    <environment name="INCLUDE" default="$HERWIGPP_BASE/include"/>
+    <environment name="LIBDIR" default="$HERWIGPP_BASE/lib/Herwig7"/>
+    <environment name="INCLUDE" default="$HERWIGPP_BASE/include/Herwig7"/>
```

Comment:
this is your original commit where you had Herwig7. The fix is part of another commit. If you see the final changes (by going to File Changed tab then you will see the right path)